scripts/testacc-dev

```diff
@@ -33,4 +33,5 @@ TF_ACC=1 \
   PREFECT_API_URL=$(op read "${vault_entry}/PREFECT_API_URL") \
   PREFECT_API_KEY=$(op read "${vault_entry}/PREFECT_API_KEY") \
   PREFECT_CLOUD_ACCOUNT_ID=$(op read "${vault_entry}/PREFECT_CLOUD_ACCOUNT_ID") \
-  gotestsum --max-fails=50 ./... -count=1 -v ${run_arg}
+  go test ./... -v ${run_arg} -sweep=foo
```

Comment:
im guessing `-sweep=all` didn't work? or is `all` supposed to be an arbitrary selector?

Reply:
Looks like `sweep` specifies the provider to sweep, and `sweep-run` lets you specify specific sweepers. Also learned that providing the `sweep` flag _only_ runs the sweepers, and that it's often recommended to:

1. Run the sweepers (for a clean slate)
2. Run the acceptance tests
3. Run the sweepers again